wiki/Tumblr/en.md

```diff
@@ -0,0 +1,12 @@
+# Tumblr
+
+*See also: [Twitter](/wiki/Twitter) and [Discord servers](/wiki/Community/Discord_servers)*
+
+This article lists all the [Tumblr](https://tumblr.com/) blogs that osu! uses to maintain and support the game's operations.
+
+| Avatar | Name | Manager | Description |
+| :-: | :-- | :-- | :-- |
+| ![](img/avatar_osunews.png) | [osu!news](https://osunews.tumblr.com/) | ::{ flag=AU }:: [osu!team](https://osu.ppy.sh/users/4341397) | Content hub feeding news updates to the main page of the osu! website. Discontinued in 2017 in favor of the [osu-wiki GitHub repository](https://github.com/ppy/osu-wiki/). |
+| ![](img/avatar_osunext.png) | [osu!next](https://osunext.tumblr.com/) | ::{ flag=MY }:: [flyte](https://osu.ppy.sh/users/3103765) | Updates on [osu!(lazer)](/wiki/Client/Release_stream/Lazer) and the osu! website. Discontinued in 2017 in favor of the [changelog page](https://osu.ppy.sh/home/changelog) and the ["lazer updates"](/wiki/Community/Video_series/lazer_updates) video series. |
+| ![](img/avatar_osustream.png) | [osu!stream](https://osustream.tumblr.com/) | ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2) | Updates on the [osu!stream](/wiki/osu!stream) project. Discontinued in 2020 after the [final release](https://blog.ppy.sh/osu-stream-2020-release/) of the app. |
+| ![](img/avatar_osuosx.png) | [osu! on OS X](https://osuosx.tumblr.com/) | ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2) | Updates on the [macOS-compatible version of osu!](/wiki/Client/Installation/macOS). |
```

Comment:
Questionable if you can still call this maintained, considering the last post was in 2018. Fine I guess 🤷